backend/app/training/scrape_products.py

```
@@ -251,18 +252,54 @@ async def scrape_products(sitemap_url, scrape_limit=float('inf'), sample_size=5,
-        # Since we are now in an async context, use asyncio.run only if running standalone.
-        products = await scrape_products(
-            first_sitemap,
-            scrape_limit=scrape_limit,
-            sample_size=sample_size,
-            max_concurrent=max_concurrent
-        )
-        return products
-    else:
-        print(f"No sitemap URLs found for {website_url}.")
-        return []
+async def scrape_from_website(
```

Ruff (0.8.2)

Missing return type annotation for public function `scrape_from_website`

(ANN201)